### PR TITLE
Setup project defaults

### PR DESCRIPTION
**Pull request by skabashnyuk**
- CI scripts
- License
- github configuration.

### CODE REVIEW

**Comment by skabashnyuk** [comment]
CC @nickboldt FYI

File: .ci/cico_common.sh [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+         bash $(pwd)/${image_dir}/build.sh --tag:${TAG} 
+         if [[ ${image_dir} == "dockerfiles/che" ]]; then
+           #CENTOS SINGLE USER
+           BUILD_ASSEMBLY_DIR=$(echo assembly/assembly-main/target/eclipse-che-*/eclipse-che-*/)
+           LOCAL_ASSEMBLY_DIR="${image_dir}/eclipse-che"
+           if [[ -d "${LOCAL_ASSEMBLY_DIR}" ]]; then
+               rm -r "${LOCAL_ASSEMBLY_DIR}"
+           fi
+           cp -r "${BUILD_ASSEMBLY_DIR}" "${LOCAL_ASSEMBLY_DIR}"
+           docker build -t ${REGISTRY}/${ORGANIZATION}/che-server:${TAG}-centos -f $(pwd)/${image_dir}/Dockerfile.centos $(pwd)/${image_dir}/
```
Comment (sleshchenko) [inline .ci/cico_common.sh]: Seems these scripts need to be updated since dashboard should not build che-server image.

Comment (skabashnyuk) [reply .ci/cico_common.sh]: done

File: .ci/cico_build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2017 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+# Just a script to get and build eclipse-che locally
+# please send PRs to github.com/kbsingh/build-run-che
```
Comment (sleshchenko) [inline .ci/cico_build.sh]: Douh.... Seems we have a lot of copy-paste which should be revised...
But it's OK to merge some working state and create a separate issue to clean scripts up

**Pull request merged**